plugin/src/main/kotlin/com/exactpro/th2/gradle/BaseTh2Plugin.kt

```diff
@@ -74,6 +75,9 @@ class BaseTh2Plugin : Plugin<Project> {
     }
 
     private fun configureOwasp(project: Project) {
+        check(GradleVersion.current() >= GradleVersion.version("8.4")) {
+            "Gradle '8.4' version or higher is required to use OWASP 9.1.0+ plugin"
```

Comment:
question: Have you checked that this error is reported on the Gradle version lower than 8.4?

Reply:
Unfortunately this code isn't executed. The `major version 65` problem happens before
```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'common'.
> java.util.concurrent.ExecutionException: org.gradle.api.GradleException: Failed to create Jar file /home/nikita.smirnov/.gradle/caches/jars-9/1550be8224c951358979eafa0ab52922/jackson-core-2.17.1.jar.
```